test_project/rename_forward/urls.py

```diff
@@ -8,6 +8,7 @@
 class LinkedDataView(autocomplete.Select2QuerySetView):
     def get_queryset(self):
         qs = super(LinkedDataView, self).get_queryset()
+        print(self.forwarded.get('checkbox', None))
```

Comment:
Another way of testing that's doesn't require code instrumentalization with print statements is to use a debugger. Such as pdb or ipdb, ever heard of it ? It's pretty cool imho <3